gateway/README.md

```diff
@@ -2,116 +2,185 @@
+
 ```bash
-kubectl get api hello -o yaml
-apiVersion: gateway.kyma.cx/v1alpha2
-kind: api
-metadata:
-  annotations:
-    kubectl.kubernetes.io/last-applied-configuration: |
-      {"apiVersion":"gateway.kyma.cx/v1alpha2","kind":"api","metadata":{"annotations":{},"labels":{"example":"serverless-lambda","function":"hello"},"name":"hello","namespace":"default"},"spec":{"authentication":[{"jwt":{"issuer":"https://dex.kyma.local","jwksUri":"http://dex-service.kyma-system.svc.cluster.local:5556/keys"},"type":"JWT"}],"hostname":"hello.kyma.local","service":{"name":"hello","port":8080}}}
-  clusterName: ""
-  creationTimestamp: 2018-05-29T08:55:04Z
-  generation: 0
-  labels:
-    example: serverless-lambda
-    function: hello
-  name: hello
-  namespace: default
-  resourceVersion: "9563"
-  selfLink: /apis/gateway.kyma.cx/v1alpha2/namespaces/default/apis/hello
-  uid: fa7805fe-631d-11e8-a42f-3e54503ab95f
-spec:
-  authentication:
-  - jwt:
-      issuer: https://dex.kyma.local
-      jwksUri: http://dex-service.kyma.local
-    type: JWT
-  hostname: hello.kyma.local
-  service:
-    name: hello
-    port: 8080
-status:
-  authenticationStatus:
-    code: 2
-    resource:
-      name: hello
-      uid: fa807328-631d-11e8-a42f-3e54503ab95f
-      version: "9562"
-  ingressStatus:
-    code: 2
-    resource:
-      name: hello-ing
-      uid: fa7d02e1-631d-11e8-a42f-3e54503ab95f
-      version: "9561"
+kubectl delete -f ./lambda/deployment.yaml -n <namespace>
+```
+
+## Troubleshooting
+
+Problem: I created an Api resource with authentication enabled but I receive responses with code 200 after making requests without a JWT token. <br><br> 
+Solution 1: Wait. If the cluster is under high workload it may take a while for authentication policies to apply. If you still have that problem after a few seconds look at the Solution 2. <br><br>
+Solution 2: Probably there is something wrong with JWKS URI you provided. However, if you used the default settings you can be 100% sure that it is proper. If you are using local deployment of Kyma (on minikube) and internal OIDC Identity Provider (like Dex) make sure that JwksUri is provided as FQDN and that it points directly to keys endpoint e.g. http://dex-service.kyma-system.svc.cluster.local:5556/keys as envoy sidecars need to be able to resolve a domain name to proper inside-cluster or outside-cluster IP address. <br><br>
+Solution 3: Check if the pod you've created has injected istio-proxy container. At first, do:
+``` bash
+kubectl get pods -n <namespace>
+```
+Find the pod created with deployment.yaml, then copy its name. Then do:
+``` bash
+kc get pod <pod-name> -n <namespace> -o json | jq '.spec.containers[].name'
 ```
+One of the returned strings should be "istio-proxy". If that is not true, the namespace probably doesn't have istio injection enabled. Go to **Additional prerequisites** section in this README to see how to fix that.
```

Comment:
``One of the returned strings should be the istio-proxy. If there is no such string, the Namespace probably does not have Istio injection enabled. Read the additional prerequisites at the beginning of the **Manual exposure using kubectl** section in this document to fix that.``